### DISCUSSION TITLE
Dropdown without bootstrap.js

### DISCUSSION THREAD

**Discussion by xvaara** [cdmoro/bootstrap-vue-3#324] [Show and tell]
Hey, I just published the branch that I have with "native" dropdown:

https://github.com/xvaara/bootstrap-vue-3/tree/popperjs-components

Ideas?

**Comment by mariusa** [comment]
I don't have enough experience to comment on this, hope @cdmoro & @aceofwings  join the discussion.

This removes the dependency on bootstrap.js (?), but introduces a dep on popper.js, right?

**Reply by xvaara** [reply]
@popperjs/core is included in bootstrap.js and widely used 3kb library, I think that is an acceptable dependency.

Put this is just the start. I'll do popover and tooltip at a later date when I have time. All the other components are quite lot easier to do than these.

**Comment by xvaara** [comment]
@cdmoro @aceofwings any comments? I have some time next week to work on removing bootstrap on more components, but if it's not needed I'll just remove it from the ones I'm using and keep them separate from here.
